libraries/anaconda3-solaris-0.1.2/context/solaris-build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+. /opt/conda/etc/profile.d/conda.sh
+
+cd $1
```

Comment:
A comment telling what $1 is and the fact that it should have an `environment.yml` file would be nice